layouts/vanilla.html

```diff
@@ -6,10 +6,10 @@
 		<meta name="theme-color" content="#fff1e0"/>
 		<meta name="dfp_site" content="test.next.ft" />
 		<link rel="stylesheet" href="/{{__name}}/main.css"/>
-		<title>{{#title}}{{this}} &mdash; {{/title}}Financial Times</title>
+		<title>{{#title}}{{{this}}} &mdash; {{/title}}Financial Times</title>
 		{{>next-mustard/check}}
 	</head>
-	<body>
+	<body class="o-hoverable-on">
```

Comment:
I think so, docs are a bit ambiguous. http://registry.origami.ft.com/components/o-hoverable@1.1.0 suggests you need the class first, and running the JS turns it off if appropriate. Doesn't seem to turn it on if the class isn't initially there